src/order/atoms.lean

```diff
@@ -88,6 +107,76 @@ is_simple_lattice_iff_is_simple_lattice_order_dual.1 (by apply_instance)
 
 end is_simple_lattice
 
+namespace is_simple_lattice
+
+variables [bounded_lattice α] [is_simple_lattice α]
+
+/-- A simple `bounded_lattice` is also distributive. -/
+protected def bounded_distrib_lattice : bounded_distrib_lattice α :=
+{ le_sup_inf := λ x y z, by { rcases eq_bot_or_eq_top x with rfl | rfl; simp },
+  .. (infer_instance : bounded_lattice α) }
```

Comment:
This `def` does not bring new data, so it can be an `instance`.

Reply:
This applies to all the instances below, right?

Reply:
Complete lattices and boolean algebras define more functions, so I think it doesn't apply to the instances below?